### PR TITLE
Support opening pinned maps

### PR DESCRIPTION
**Pull request by levaitamas**
Awesome project, thanks!
Made a `getMapFDFromSomewhereElse()` implementation which works with pinned maps.
```js
const fd = bpf.objGet("/sys/fs/bpf/tc/globals/....");
```

### CODE REVIEW

File: lib/map/common.ts [hunk 1]
```diff
@@ -289,3 +289,10 @@ export function checkAllProcessed(count: number | undefined, batchSize: number)
         throw Error(`Assertion failed: ${count} of ${batchSize} entries processed`)
     }
 }
+
+export function objGet(path: String): number | undefined {
+    const fd = native.bpfObjGet(path)
+    if (fd < 0)
+        return undefined
```
Comment (mildsunrise) [inline lib/map/common.ts]: Thank you for the patch :) It's mostly ready, but `checkStatus` must be used here:

```suggestion
    checkStatus('bpf_obj_get', fd)
```

**Comment by levaitamas** [comment]
Thanks for the comment. I fixed it in b888b32 and added function documentation  in 1281c1a.

**Comment by mildsunrise** [comment]
perfect. it would be good to test this, but we can't do that with the APIs we have, and CI is failing anyway... so I'll think I'll merge and release

**Comment by levaitamas** [comment]
Would you mind releasing a new version?

**Comment by mildsunrise** [comment]
sure! v1.3.0 is publishing

**Pull request merged**